vortex-expr/src/exprs/binary.rs

```diff
@@ -138,6 +138,20 @@ impl Display for BinaryExpr {
 
 impl AnalysisExpr for BinaryExpr {
```

Comment:
We can add an example transformed predicate?

Reply:
There isn't really any documentation in this file, I've updated the pruning_expr unit tests. I agree could be some more documentation here